dicebot/data/types/message_context.py

```diff
@@ -42,11 +42,20 @@ def channel(self) -> discord.TextChannel | discord.DMChannel:
         )
         return self.message.channel
 
-    async def quote_reply(self, msg: str) -> None:
+    async def send(self, msg: str, silent: bool = True) -> None:
         if len(msg) <= MAX_CHARS_PER_MSG:
-            await self.channel.send(msg, reference=self.message)
+            await self.channel.send(msg, silent=silent)
             return
```

Comment:
It would definitely be nice if `send` had the same signature as `[channel.send](https://discordpy.readthedocs.io/en/stable/api.html?highlight=channel%20send#discord.TextChannel.send)`. Since that one is fairly complicated, let's do this instead:
```python3
async def send(self, *args, silent: bool = True, **kwargs) -> None:
    # Other logic around chunking and such
    await self.channel.send(*args, silent=silent, **kwargs)
```

Reply:
Ah nice, can do.